site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+| Toloka    | Toloka, developed by Yandex, is a crowdsourcing platform that allows users to assign tasks to a broad group of participants, often termed "crowd workers". In the context of this article, when we mention Toloka, we are specifically referring to one of its UI interfaces. |
+| CVAT      | CVAT, developed by OpenCV, is a visualization tool designed for annotating video and image data for computer vision tasks. In this article's context, when we reference CVAT, we mean one of its UI interfaces.                                                               |
+| Requester | CVAT and Toloka user. An individual who establishes an annotation project within CVAT determines the price, creates tasks and jobs within the project, and then releases it on Toloka.                                                                                        |
+| Toloker   | Toloka user. A person who annotates the Requester's dataset using the Toloka interface.                                                                                                                                                                                       |
+
+<!--lint enable maximum-line-length-->
+
+## Preconditions
+
+Actor: Requester.
+
+Requester must have a [CVAT account](/docs/manual/basics/registration/)
+and [Toloka Requester](https://platform.toloka.ai/auth?role=REQUESTER&retpath=%2Fsignup%2Frequester)
+account.
+
+## Creating Toloka project
+
+The requester can set up a project within CVAT and subsequently
+connect it to Toloka, making it accessible for annotations by Tolokers.
+
+To initiate your Toloka project, proceed with the following steps:
+
+1. [Log in to CVAT](/docs/manual/basics/registration/#account-access)
+   and [initiate a new Project](/docs/manual/advanced/projects/).
+2. Navigate to the project page and select **Actions** > **Setup crowdsourcing project**.
+
+   ![Setting Up Toloka Project](/images/toloka01.png)
+
+3. Fill in the following fields in the **Setup crowdsourcing project** form:
+
+   ![Toloka Project Configuration](/images/toloka02.png)
+
+   - **Provider**: Choose Toloka as your provider.
+   - **Environment**: Select either **Sandbox** for a testing environment or **Production** for a live environment.
+   - **Authorization key**: Enter the [Requester Authentication Key](https://toloka.ai/docs/toloka-kit/registration/).
+   - **Project description** (Optional): Provide a brief description of your project.
+
+4. Click **Submit**.
+   A pop-up indicating **Toloka Project Created** will appear, along with the **Update project** form.
+
+   ![Project Update Form](/images/toloka03.png)
+
+   - ![Open Project in Toloka](/images/toloka04.png) will take you to the project in Toloka.
+   - ![Open Project in CVAT](/images/toloka05.png) will take you back to the project in CVAT.
+
+In CVAT, all projects related to Toloka will be labeled as **Toloka**.
+
+![Toloka Label on Project](/images/toloka06.png)
```

Comment:
Could you please also add information about the status indicator, which is located on the right side of the Toloka icon? For projects: green - active, dark grey - archived.
For pools: green - open for annotating, light gray - closed, dark grey - archived.